### DISCUSSION TITLE
Providence of range_min & range_max

### DISCUSSION THREAD

**Discussion by purcell-lab** [amberelectric/public-api#150] [General]
I see that (sometimes) the API returns min and max ranges for price forecasts.

Are these values useful and where do they come from?

Does Amber synthetically create these from AEMO data or are these provided by AEMO directly?

Why do these min/ max values sometimes disappear?

![Screenshot_20230728-212049](https://github.com/amberelectric/public-api/assets/79175134/18401d06-dec6-4fc4-9c9e-3a39006d1332)

**Comment by madpilot** [comment]
AEMO use around a 100 different models to form the forecast. The min and max are the smallest and largest values from all the results. It's a crude volatility metric. If the min and max are the same as the forecasted value, then they are omitted from the response.

**Comment by purcell-lab** [comment]
Thanks for the explainer..
